components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/java/org/wso2/carbon/identity/api/server/application/management/v1/core/ServerApplicationManagementService.java

```diff
@@ -283,10 +295,10 @@ public ApplicationListResponse getAllApplications(Integer limit, Integer offset,
     private void validateRequiredAttributes(List<String> requestedAttributeList) {
 
         for (String attribute: requestedAttributeList) {
-            /* 'attributes' requested in get application list only supports advancedConfigurations and templateId.
+            /* 'attributes' requested in get applications only supports advancedConfigurations, templateId, clientId.
             * The advancedConfigurations is supported as metadata of the application is essential in certain cases and
-            * templateId is supported to add filtering on listing page. */
-            if (!(attribute.equals(ADVANCED_CONFIGURATIONS) || attribute.equals(TEMPLATE_ID))) {
+            * templateId, and clientId attributes are supported to add filtering on listing page. */
```

Comment:
Fix this comment because templateId is not supported in filter application feature.

Reply:
If the last part is not required in the comment, we can remove that